union-ai-admin/aws/script/generate.py

```diff
@@ -329,6 +329,9 @@ def create_updater_policy(role_type):
                         Action("ec2", "ModifyLaunchTemplate"),
                     ],
                     Resource=[
+                        Sub(
+                            "arn:aws:ec2:${AWS::Region}:${AWS::AccountId}:vpc-endpoint/*"
+                        ),
```

Comment:
Did you intend to add this?  The PR title says add missing S3 permissions, but this is related to VPC endpoints.